src/DDserver.cpp

```diff
@@ -187,72 +186,72 @@ namespace DoubleD
 
         CROW_ROUTE(app, "/status")
             ([&](const crow::request& req) {
+            
             crow::json::wvalue x;
+            x["servername"] = DDserver::m_server_name;
 
             if (req.url_params.get("auth") == nullptr)
             {
-                x[DDserver::m_server_name] = "no key";
+                x["status"] = "no api key";
                 return crow::response(400, x);
             }
             else if (DDserver::m_api_key != req.url_params.get("auth"))
             {
-                x[DDserver::m_server_name] = "invalid key";
+                x["status"] = "invalid key";
                 return crow::response(401, x);
             }
 
+            x["status"] = "ok";
+
+            // Create an empty vector.
+            // Crow will convert this to a JSON list.
+            // The list will be overwritten if the server contains locks.
+            std::vector<std::string> jsonList;
+            x["locks"] = jsonList;
```

Comment:
This will make sure that the json always contains a list, even when there are no locks.